crates/containerd-shim-wasm/Cargo.toml

```diff
@@ -20,18 +20,21 @@ crossbeam = { workspace = true }
 libc = { workspace = true }
 log = { workspace = true }
 oci-spec = { workspace = true }
+artifact-spec = { package = "oci-spec", git="https://github.com/containers/oci-spec-rs", rev="c109fe19b2548a86f20e48b18831bf17eb72c494", features = ["runtime"] }
```

Comment:
I believe this will prevent us from being able to publish crates since it is not a crate dependency.

Reply:
One thing I could do is just parse the fields required to do the work and only use this commit in the test image projects but I did open https://github.com/containers/oci-spec-rs/pull/147

Reply:
https://github.com/containerd/runwasi/pull/352

Reply:
updated